README.md

```diff
@@ -36,10 +36,17 @@ import alias from 'rollup-plugin-alias';
 export default {
   input: './src/index.js',
   plugins: [alias({
-    somelibrary: './mylocallibrary'
+    resolve: ['.jsx', '.js'],
```

Comment:
Is this now mandatory? Below in the text the array is described as optional. Maybe we could still have a short-hand syntax then, which would be

`alias([{find: '...', replacement: '...'}])`

i.e. you can just provide an array of replacements instead of an object?

Reply:
No, it's not manadatory. I did aim to introduce as little changes to the logic as possible with the initial change, that's why I didn't touch the resolve option (it's still handled seperately).